src/main/java/io/reactivex/disposables/CompositeDisposable.java

```diff
@@ -223,7 +225,14 @@ void dispose(OpenHashSet<Disposable> set) {
         }
         if (errors != null) {
             if (errors.size() == 1) {
-                throw Exceptions.propagate(errors.get(0));
+                Throwable ex = errors.get(0);
+                if (ex instanceof Error) {
+                    throw (Error)ex;
+                }
+                if (ex instanceof RuntimeException) {
```

Comment:
The reason I inlined these here is because if I'm using `propagate`, that shows up in the code coverage analysis: the `throw Exceptions.propagate(ex)` never returns properly and thus the coverage thinks a code path is not covered. It is usually the last roadblock to a 100% covered file. I've been thinking about having `ExceptionHelper.wrapOrThrow()` that throws for non-checked and returns a wrapped checked that will run through throw.